contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
OK good. It's been about a year since I've done Fedora packaging. Do you see a need to depend on the official packages for Dockerfile based builds? Do the guidelines specify preference of the official packages for security purposes?